pkgs/development/libraries/libgpiod/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, autoconf, autoconf-archive, automake, libtool, pkgconfig, kmod, enable-tools ? true }:
+
+stdenv.mkDerivation rec {
+  name = "libgpiod-${version}";
+  version = "2018-10-07";
+
+  src = fetchgit {
+    url = "git://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git";
```

this one also works in ugly networks:

```suggestion
    url = "https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git";
```